pals/mysterious-halfling.md

```diff
@@ -0,0 +1,65 @@
+
+# ??? It Is A Mystery
+
+> ??? This Is Also A Mystery
+
+#### Species, Class, Level:
+Halfing Rogue, Level 1
+
+#### Languages known:
+Common, +1 other
+
+#### Attributes
+* Str: 16 +3
+* Con: 12 +1
+* Dex: 16 +3
+* Int: 11 -
+* Wis: 8  -1
+* Cha: 16 +3
+
+#### Defense scores
+* AC: 15
+* Fort: 13
+* Ref: 15
+* Will: 13
+
+#### Other scores
+* Initiative +3
+* Speed 6
+* Hit Points: 24
+* Surge Value: 6
+* Surges: 7
+
+#### Powers
+* Bold
+* Nimble Reaction
+* First strike
+* Rogue Tactic: Brutal Scoundrel
+* Rogue Weapon Talent
+* Sneak Attack
+* Riposte Strike (At-will)
+* Piercing Strike (At-will)
+* Torturous Strike (Encounter)
+* Second Chance (Encounter)
+* Easy Target (Daily)
+
+#### Trained Skills
+
+* acrobatics +5
+* bluff +8
+* insight +4
+* perception +4
+* stealth +8
+* thievery +10
+
+#### Feats
+* Lost in the Crowd
+
+#### Equipment
+* Leather Armor
+* 5x Dagger
+* 10x Shuriken
+* 3x Torches
+* Adventurer's Kit
+* Thieves' tools
```

Comment:
Yes, it's like matryoshka dolls, you release a smaller thief into the lock and they have their own heist adventure to break it.